BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -0,0 +1,988 @@
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Max drift</color> - maximum allowed angle between facing and velocity vector");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Moving pitch</color> - the pitch level to maintain when moving at cruise speed");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Bank angle</color> - the limit on roll when turning, positive rolls into turns");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Steer Factor</color> - higher will make the AI apply more control input for the same desired rotation");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Steer Damping</color> - higher will make the AI apply more control input when it wants to stop rotation");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Attack vector</color> - does the vessel attack from the front or the sides");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Min engagement range</color> - AI will try to move away from oponents if closer than this range");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Max engagement range</color> - AI will prioritize getting closer over attacking when beyond this range");
+            sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- RCS active</color> - Use RCS during any maneuvers, or only in combat ");
+            if (GameSettings.ADVANCED_TWEAKABLES)
+            {
+                sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Min obstacle mass</color> - Obstacles of a lower mass than this will be ignored instead of avoided");
+                sb.AppendLine($"<color={XKCDColors.HexFormat.Cyan}>- Goes up to</color> - Increases variable limits, no direct effect on behaviour");
+            }
+
+            return sb.ToString();
+        }
+
+        #endregion RMB info in editor
+
+        #region events
+
+        public override void OnStart(StartState state)
+        {
+            base.OnStart(state);
+        }
+
+        public override void ActivatePilot()
+        {
+            base.ActivatePilot();
+
+            if (!fc)
+            {
+                fc = gameObject.AddComponent<BDOrbitalControl>();
+                fc.vessel = vessel;
+
+                fc.alignmentToleranceforBurn = 7.5f;
+                fc.throttleLerpRate = 3;
+            }
+            fc.Activate();
+            updateInterval = combatUpdateInterval;
+
+            if (maxAccelerationCR == null) maxAccelerationCR = StartCoroutine(CalculateMaxAcceleration());
+            if (shipController == null) shipController = StartCoroutine(ShipController());
+        }
+
+        public override void DeactivatePilot()
+        {
+            base.DeactivatePilot();
+
+            if (fc)
+                fc.Deactivate();
+            
+            if (maxAccelerationCR != null) StopCoroutine(maxAccelerationCR);
```

Comment:
These need to set `maxAccelerationCR` and `shipController` to `null` after they're stopped.